benches/profile/collect_worker_shadow.py

```
@@ -0,0 +1,166 @@
+
+
+def order(batch_index):
+    return ("disabled", "shadow") if batch_index % 2 == 0 else ("shadow", "disabled")
+
+
+def parse_comparisons(output):
+    matches = COMPARISON_RE.findall(output)
+    if len(matches) != 1:
+        raise ValueError(f"expected one complete shadow summary, got {len(matches)}")
+    values = map(int, matches[0])
+    result = dict(zip(("matched", "mismatched", "superseded", "pending"), values))
```

Ruff (0.15.21)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)